pkgs/applications/science/electronics/librepcb/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchgit, qtbase, qttools, qmake, mesa, openssl, zlib }:
+
+stdenv.mkDerivation rec {
+	name = "librepcb-unstable";
```

Comment:
the name should contain the version or a date of last commit, if a software project does make releases